versions.txt

```diff
@@ -123,3 +123,6 @@
 1.7.6.4
 1.7.6.5
 1.7.6.7
+1.7.6.6
+1.7.6.7
+1.7.7.0-beta.2
```

Comment:
How come we add the beta build this time? Seems like we never did it for previous versions 🤔

Reply:
It creates the Dockerfile for the beta version, but we must push it manually for the moment.
I plan to refactor this repository in order to manage property PHP / PrestaShop version.

Reply:
Mmm, but we didn't push any other beta, so why this one?
Unless we clean them when the official release is ready?

Reply:
It's a new demand from the community, they created an issue weeks ago. We will do it for all our versions, beta, alpha, RC, etc.

Reply:
So it's in addition with the already existing beta build? This one is built on the branch last nighly build though so I guess the community needs a Docker that matches the beta tag?

Reply:
https://github.com/PrestaShop/PrestaShop/issues/20563 They are also able to use the nightly build, but for testing usage, it's also better to have access to the same beta build we pushed on the dot com.

Reply:
Ok nice improvement indeed! I'll just add the issue in the description to understand the new need

Also we might need to update the release doc, because now we will have to update the docker images for each pre-release build